sdk/trace/id_generator_test.go

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package trace
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func TestNewIDs(t *testing.T) {
+	gen := defaultIDGenerator()
+
+	for i := 0; i < 1000; i++ {
+		traceID, spanID := gen.NewIDs(context.Background())
+		assert.True(t, traceID.IsValid())
+		assert.True(t, spanID.IsValid())
+	}
+}
+
+func TestNewSpanID(t *testing.T) {
+	gen := defaultIDGenerator()
+	traceId, err := trace.TraceIDFromHex("4bf92f3577b34da6a3ce929d0e0e4736")
+	require.NoError(t, err)
+
+	for i := 0; i < 1000; i++ {
```

Comment:
In that case, if it fails, we should provide more meaningful information, at least the `spanID` which failed. Otherwise, we'd end up with a failing test that we know nothing about.

Reply:
Good point! Added.
Now it looks like below in screenshot when any spanID is failed.
<img width="882" alt="Screenshot 2023-05-03 at 1 10 45 AM" src="https://user-images.githubusercontent.com/22260860/235864204-59f3219d-c8b2-464b-aed4-70663f76453c.png">